### PR TITLE
Rodrigo sphinx

### PR DESCRIPTION
**Pull request by roornelas**

### CODE REVIEW

**Comment by roornelas** [comment]
@anoshaminai @sandlerben 
Please try getting the large files (see readme) and then running these commands. I think you do not even need to have gradle installed because I set it up with a wrapper

files:
https://onedrive.live.com/redir?resid=427D6D69FE5B4BA3!40133&authkey=!AFIK0llMvpxykPk&ithint=file%2czip

./gradlew build
./gradlew run

**Comment by sandlerben** [comment]
Sorry I haven't looked this over yet, the beginning of my week is a little
crazy. I'll take a look Wednesday or Thursday.

## 

Benjamin Sandler
University of Pennsylvania | Class of 2018
Candidate for BS in Computer Science & BA in Cognitive Science
954.478.6689 | ben1sandler@gmail.com

**Comment by sandlerben** [comment]
@roornelas I merged in master to try to get tests passing

File: Sphinx/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Missing:
```
Comment (sandlerben) [inline Sphinx/README.md]: Do you want to put this in the top-level README? If not, we should at least link to this README from the top-level one.

File: Sphinx/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+cmusphinx-en-us-5.2
+en-us.lm
+
+Get from this link:
+
+https://onedrive.live.com/redir?resid=427D6D69FE5B4BA3!40133&authkey=!AFIK0llMvpxykPk&ithint=file%2czip
+
+Or get files from official source and rename:
```
Comment (sandlerben) [inline Sphinx/README.md]: Let's just include instructions on how to get them from official source. What do you mean by rename? Can you make the instructions a little bit more clear?

Do they need to be renamed to `cmusphinx-en-us-5.2` and `en-us.lm`? If so, let's change the first name to `cmusphinx-en-us`.

File: Sphinx/build.gradle [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    testCompile 'junit:junit:4.12'
+    compile group: 'edu.cmu.sphinx', name: 'sphinx4-core', version:'5prealpha-SNAPSHOT'
+    compile group: 'edu.cmu.sphinx', name: 'sphinx4-data', version:'5prealpha-SNAPSHOT'
+    compile 'com.google.code.gson:gson:2.6.2'
+
+
+}
+apply plugin: 'application'
+
+mainClassName = 'TranscriberDemo'
```
Comment (sandlerben) [inline Sphinx/build.gradle]: Can we get rid of the word `demo` everywhere?

**Comment by sandlerben** [comment]
Can we make the sphinx code take in the file to transcribe as an argument?

**Comment by anoshaminai** [comment]
@roornelas The commands work for me and I did not have gradle downloaded beforehand!

File: README.md [hunk 4]
```diff
@@ -29,6 +29,12 @@ $ go build
 $ ./transcribe4all
 ```
 
+
+## Sphinx
```
Comment (sandlerben) [inline README.md]: Make this part a subheading of the "Go Set Up" section

File: README.md [hunk 5]
```diff
@@ -29,6 +29,12 @@ $ go build
...
+
+If you are going to use Sphinx for transcription look at the following [instructions](Sphinx/README.md)
```
Comment (sandlerben) [inline README.md]: Change to "To set up Sphinx for transcription read the following instructions"

File: Sphinx/README.md [hunk 6]
```diff
@@ -1,19 +1,21 @@
...
-en-us.lm
 
-Get from this link:
+Acoustic Model- cmusphinx-en-us
 
-https://onedrive.live.com/redir?resid=427D6D69FE5B4BA3!40133&authkey=!AFIK0llMvpxykPk&ithint=file%2czip
+Language Model- en-us.lm
 
-Or get files from official source and rename:
+Get from both files from this link:
```
Comment (sandlerben) [inline Sphinx/README.md]: Reword

File: Sphinx/README.md [hunk 7]
```diff
@@ -1,19 +1,21 @@
 Missing:
```
Comment (sandlerben) [inline Sphinx/README.md]: Can you make this file a little prettier? 
- Use Markdown headers
- Use [hyperlinks](http://yes)

**Comment by sandlerben** [comment]
Looks good to me after the last round of fixes

**Comment by roornelas** [comment]
Passing filename as an argument is now supported:

$ ./gradlew run '-Pmyargs=files/Insider'

File: Sphinx/README.md [hunk 8]
```diff
@@ -1,21 +1,29 @@
...
-cmusphinx-en-us-5.2 (or most recent one) to cmusphinx-en-us
+[cmusphinx-en-us-5.2](https://sourceforge.net/projects/cmusphinx/files/Acoustic%20and%20Language%20Models/US%20English%20Generic%20Acoustic%20Model/) 
 
-https://sourceforge.net/projects/cmusphinx/files/Acoustic%20and%20Language%20Models/US%20English%20Generic%20Language%20Model/
+Rename file to cmusphinx-en-us
 
-cmusphinx-5.0-en-us.lm (or most recent one) to  en-us.lm
+
+
+*Language Model*
```
Comment (sandlerben) [inline Sphinx/README.md:23]: lowercase l in model?

Comment (sandlerben) [inline Sphinx/README.md:23]: or maybe that is lowercase and just looks weird

**Comment by sandlerben** [comment]
@roornelas please try to get this merged today or tomorrow

**Pull request merged**